ivy/functional/frontends/torch/tensor.py

```diff
@@ -525,6 +525,11 @@ def flatten(self, start_dim, end_dim):
     def cumsum(self, dim, dtype):
         return torch_frontend.cumsum(self._ivy_array, dim, dtype=dtype)
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
+    def cumsum_(self, dim, dtype):
```

Comment:
`dtype` should have `None` as the default value (https://pytorch.org/docs/1.11/generated/torch.Tensor.cumsum_.html?highlight=cumsum#torch.Tensor.cumsum_)